BT-Tracking/Interface/Data List/DataCollectionInfoVC.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  DataCollectionInfoVC.swift
+//  BT-Tracking
+//
+//  Created by Michal Šrůtek on 14/04/2020.
+//  Copyright © 2020 Covid19CZ. All rights reserved.
+//
+
+import UIKit
+import MarkdownKit
+
+final class DataCollectionInfoVC: UIViewController {
+
+    // MARK: Private Properties
+    private let textView = UITextView()
+
+    // MARK: Lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        initViews()
+        layoutViews()
+        setupContent()
+    }
+
+    // MARK: Setup
+
+    private func initViews() {
+        title = "Informace o sběru dat"
+
+        navigationItem.largeTitleDisplayMode = .never
+        edgesForExtendedLayout = []
+
+        textView.textContainerInset = UIEdgeInsets(top: 30, left: 16, bottom: 16, right: 16)
```

Comment:
You need to set correct right and left margins for iPhone plus size and iPads

Reply:
Got inspired in `HelpVC`. The constraints and edge insets seem to be the same in there. What would you suggest? @augard

Reply:
I wasn't available when help was merged 😬. So you need to update it from controller.view layoutMargins. And there is override pint in controller "viewLayoutMarginsDidChange" when it's changed.

Reply:
Can you please change it also in helpvc, it should be about 20px for plus phone or iPad, and 16 for other devices.

Reply:
@augard Updated. `HelpVC` seems to be doing that by storyboard-ly.